server/build/replace.js

```diff
@@ -7,10 +7,15 @@ export default async function replaceCurrentBuild (dir, buildDir) {
   const oldDir = join(buildDir, '.next.old')
 
   try {
-    await rename(_dir, oldDir)
+    await move(_dir, oldDir)
```

Comment:
The `mv` package doesn't return a promise but rather expects a callback, in this case await will not wait for the move to complete.
https://github.com/andrewrk/node-mv/blob/master/index.js